tensorflow/lite/c/CMakeLists.txt

```diff
@@ -0,0 +1,57 @@
+cmake_minimum_required(VERSION 3.16)
+project(tensorflow-lite-c C CXX)
+
+option(TFLITE_C_BUILD_SHARED_LIBS "Build shared libraries" ON)
+
+set(TENSORFLOW_SOURCE_DIR "" CACHE PATH
+  "Directory that contains the TensorFlow project"
+)
+if (NOT TENSORFLOW_SOURCE_DIR)
+  get_filename_component(TENSORFLOW_SOURCE_DIR
+    "${CMAKE_CURRENT_LIST_DIR}/../../../"
+    ABSOLUTE
+  )
+endif()
+
+add_subdirectory(
+  "${TENSORFLOW_SOURCE_DIR}/tensorflow/lite"
+  "${CMAKE_CURRENT_BINARY_DIR}/tensorflow-lite"
+  EXCLUDE_FROM_ALL
+)
+
+set(CMAKE_CXX_STANDARD 11)
+
+set(TFLITE_C_LIBTYPE STATIC)
+if (TFLITE_C_BUILD_SHARED_LIBS)
+  set(TFLITE_C_LIBTYPE SHARED)
+endif()
+
+add_library(tensorflowlite_c ${TFLITE_C_LIBTYPE}
+  builtin_op_data.h
+  common.h
+  common.c
+  c_api_types.h
+  c_api.h
+  c_api.cc
+  c_api_experimental.h
+  c_api_experimental.cc
+  c_api_internal.h
+)
+
+if (TFLITE_C_BUILD_SHARED_LIBS)
+  if (UNIX)
```

Comment:
Can't we remove UNIX?

if (APPLE)
elseif (WIN32)
else()
endif()

Reply:
Seems better.